about.html

```diff
@@ -0,0 +1,182 @@
+          </div>
+          <h1 class="head-motto">
+            <q class="motto-quot">Irreechaa  Love,Peace and Unity</q>
+         <span class="motto-clip">
+          CELEBRATING Erriecha<br> FESTIVAL 2023
+          <br>
+         </span>
+          </h1>
+          <p class="head-paragraph txt-second bg-white">
+            Irreechaa is usually celebrated during the weekend following the Meskel holiday. The festival is observed in many areas of Oromia, but, at the national level, it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital. 
+            <span class="desktop-only">
+              This year we hope to expand our invitation to individuals and organizations including world class Medias to promote this beautiful culture to the rest of the world.<br>
+              By doing this we made the celebration festival to be  the source of tourist attraction. So if you are interested to be part of celebration we hope  you will consider join us this year to make Irreecha festival a world class celebrated festival.
+            </span>
+
+          </p>
+          
+          <p class="par-contact">
+            Please contact as per email for any further question about the 2023 Irreechaa Celebration Festival <a href="#">irrechafestival2023@otc.org</a>
+          </p>
+          <div class="bg-white">
+          <h3 class="logo-design">
+            Design of Erriecha Festival 2023 Logo
+          </h3>
+          <p class="logo-designer">
+            The logo of Erriecha Was designed by Osman Wako Full Stack Software Developer.
+          </p>
+          <div class="festival-logo">
+            <img src="./assets/images/titleicon.png" alt="Irreechaa Logo" class="festival-images">
+          </div>
+        <hr>
+        </div>
+        </div>
+      </section>
+      <section>
+        <div class="start-align end-align">
+          <h1 class="seepastyear"><br>See The Past Erriecha Festival</h1>
+        <p class="par-history seepastyear">
+          Take a look at the recent two Erriecha Festivals which took place in Hora Finfinne and Hora Har-sadii.
+        </p>
+        <div class="grid-photo">
+          <div class="grid-item grid-album-1">
+          <img src="./assets/images/festival/fest1.jpg" class="fest-img" >
+            <p class="overlay-text">2022<br>Erriecha Annual Festival</p>
+          </div>
+          <div class="grid-item grid-album-1">
+            <img src="./assets/images/festival/fest3.jpg" class="fest-img">
+            <p class="overlay-text">2020<br>Erriecha Annual Festival</p>
+          </div>
+          <div class="grid-item grid-album-1">
+            <img src="./assets/images/festival/fest2.png" class="fest-img">
+            <p class="overlay-text">2019<br>Erriecha Annual Festival</p>
+          </div>
+        </div>
+        </div>
+      </section>
+    </main>
```

Comment:
- [ ] Kindly ensure that your design is in line with the guiding template. At this moment, your design is completely different from the guiding template. Please, check the guiding template and walk your design alongside the design template.

### Your Design
![your-about-image](https://user-images.githubusercontent.com/26597432/226097096-b8c42c6f-52eb-4bcd-a394-4124d49cb001.png)

### Guiding Template
![temp-about-image](https://user-images.githubusercontent.com/26597432/226097102-5d245679-0135-4803-8a31-b0daaba4fcfa.png)